pyswmm/simulation.py

```diff
@@ -83,7 +85,8 @@ def __init__(self,
                  inputfile,
                  reportfile=None,
                  outputfile=None,
-                 sim_preconfig=None):
+                 sim_preconfig=None,
+                 suppress_context_warning=False):
```

Comment:
Can we remove the `suppress_context_warning` parameter since it is no longer used?



```suggestion
                 outputfile=None,
                 sim_preconfig=None):
```